pallets/lbp/src/benchmarking.rs

```diff
@@ -36,6 +36,8 @@ const ASSET_B_AMOUNT: Balance = 2_000_000_000;
 const INITIAL_WEIGHT: LBPWeight = 20_000_000;
 const FINAL_WEIGHT: LBPWeight = 90_000_000;
 
+const DEFAULT_FEE: Fee = Fee {numerator: 2, denominator: 1_000};
```

Comment:
LBP pallet does not have configurable fee ?

Reply:
Yes, fees are configurable, but not in the config. The fee is set during pool creation.